src/lib/datepicker/datepicker-input.ts

```diff
@@ -53,6 +57,9 @@ export const MAT_DATEPICKER_VALIDATORS: any = {
 
 
 /**
+ * @deprecated
+ * @deletion-target 8.0.0 Use `DatepickerInputEvent<D>` instead.
```

Comment:
The deprecation messages should be on the `@deprecated` tag, instead of the `@deletion-target`.